docs/source/testing/testing.md

```diff
@@ -3,26 +3,55 @@ title: Integration testing
 description: Utilities for testing Apollo Server
 ---
 
-Testing `apollo-server` can be done in many ways. One simple way is to use ApolloServer's `executeOperation` method to directly execute a GraphQL operation without going through a full HTTP operation.
-
-## `executeOperation`
-
-Integration testing a GraphQL server means testing many things. `apollo-server` has a request pipeline that can support many plugins that can affect the way an operation is executed. The `executeOperation` method provides a single hook to run operations through the request pipeline, enabling the most thorough tests possible without starting up an HTTP server.
+Integration testing a GraphQL server means testing the combination of many interlocking parts at once. One way to test your server setup is to use `ApolloServer`'s `executeOperation` method to directly execute a GraphQL operation without going through a full HTTP operation.
+
+## Testing using `executeOperation`
+
+The `apollo-server` library has a request pipeline that can support many plugins which can each affect the way an operation is executed. The `executeOperation` method is available to run operations through the request pipeline, enabling the most thorough tests possible without starting up an HTTP server. 
+
+The [`executeOperation` method](../api/apollo-server/#executeoperation) can accept the following arguments: 
+* An object containing configuration options which must include a `query` key specifying the GraphQL operation to be run. 
```

Comment:
"options which" -> "options, which" (always comma before 'which', never comma before 'that')

"`query` key specifying" -> "`query` field that specifies" (I try not to use 'key' in situations I can avoid that aren't related to an object's _identifier_)

"to be run" -> "to run" (avoid passive where comfortably avoidable)